pkg/kopia/command/storage/secret_utils.go

```diff
@@ -183,6 +184,9 @@ func GetMapForLocationValues(
 	}
 	if locType != "" {
 		m[typeKey] = []byte(locType)
+		if locType == LocType(v1alpha1.LocationTypeS3Compliant) {
+			m[typeKey] = []byte(LocTypeS3)
+		}
```

Comment:
what is the value of `locType` in case of the if statement? is it not LocTypeS3 already?

LGTM

Reply:
For Kanister profiles, it's `s3Compliant` while we have `s3` as value of LocTypeS3.